src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -80,15 +80,15 @@ public void WriteAssembly(IAssemblySymbol assemblySymbol)
             SyntaxNode compilationUnit = _syntaxGenerator.CompilationUnit(namespaceSyntaxNodes)
                 .WithAdditionalAnnotations(Formatter.Annotation, Simplifier.Annotation)
                 .Rewrite(new TypeDeclarationCSharpSyntaxRewriter())
-                .Rewrite(new BodyBlockCSharpSyntaxRewriter(_exceptionMessage))
-                .NormalizeWhitespace();
```

Comment:
Why did you decide to move this call to line 83? Wouldn't adding the `eol: Environment.NewLine` argument here be enough?

Reply:
Because the CompilationUnit is mutated below. The `NormalizeWhitespace` invocation shouldn't happen before the syntax tree is finale.